lib/index.js

```diff
@@ -11,8 +11,10 @@ var integration = require('@segment/analytics.js-integration');
  */
 
 var Castle = module.exports = integration('Castle')
-  .option('appId', '')
-  .tag('<script src="//d2t77mnxyo7adj.cloudfront.net/v1/c.js">');
+  .option('publishableKey', '')
+  .option('autoPageview', true)
+  .option('cookieDomain', undefined)
```

Comment:
i think we should just use `false` as the default value here *nitpick*. 
We generally never use `undefined` as the default value if the desired effect is just to have a default falsy value